### ISSUE TITLE
Sidebar button does not have a description useful for screen reader users

### ISSUE THREAD

**Issue by andrapetronela** [docsifyjs/docsify#658]
Accessibility test errors

![screenshot 143](https://user-images.githubusercontent.com/34648710/47645961-5d1d0780-db6a-11e8-999d-15cddd9dbc51.png)

It would be nice to have an aria-label attribute that will help screen reader users understand what action will perform the button.
 Without an `aria-label` attribute:

- the sidebar button is identified by the screen readers only as a button element
- impossible to the user to know what action will perform the button, once clicked / selected

`aria-label` has no effect on browsers. It only helps assistive technology attach a label to an HTML element.

**Comment by cheng-kang** [comment]
This is definitely a thing to be added. Would be best if you could make some suggestions about the labels 😄

**Comment by andrapetronela** [comment]
- <button `aria-label="Toggle sidebar"`>
- <input `aria-label="Search through site content"`>

What do you think, @cheng-kang? :)

**Comment by chris-79** [comment]
_Bump_

We _need_ this

**Comment by jhildenbiddle** [comment]
Thanks for creating the issue and for your patience white waiting for it to be addressed, @andrapetronela.

I've created a draft PR here: #2253. In that PR you will find a preview link which will allow you to review the changes:

![CleanShot 2023-10-17 at 08 59 27@2x](https://github.com/docsifyjs/docsify/assets/442527/7e4bf479-b02c-46c3-8e98-a99ffefa1ae5)

Before I mark this PR as ready for review by the maintainers, I would love to get your thoughts and/or feedback.

Thanks!
